md2html/render_html.h

```diff
@@ -26,7 +26,11 @@
 #ifndef MD4C_RENDER_HTML_H
 #define MD4C_RENDER_HTML_H
 
-#include "md4c.h"
+#include "../md4c/md4c.h"
```

Comment:
Because of the quotes I thought this should be a private local include (since it’s an example) and the header would be itself. If this is moved into a public API, I agree that we should not force relative paths. But this PR is closed already, so no more discussion needed.